tensorflow/python/keras/distribute/distributed_training_utils.py

```diff
@@ -575,10 +575,13 @@ def _prepare_feed_values(model, inputs, targets, sample_weights, mode):
   if mode == ModeKeys.PREDICT:
     sample_weights = []
     targets = []
-  else:
+  elif sample_weights is None:
     sample_weights = [
```

Comment:
I think we need this to be inside the else condition?

Reply:
Hmm, Sorry, I'm not sure I understand what you mean above. But I have changed the logic there, please take a look at it.

Reply:
Sorry I think I did not see the latest change for some reason.

Reply:
Can you see now?